Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/RideRequestUtils.java

```diff
@@ -784,4 +795,19 @@ public static String getCurrentUTC(){
         f.setTimeZone(TimeZone.getTimeZone("UTC"));
         return f.format(new Date());
     }
+    
+    @SuppressLint("SetTextI18n")
```

Comment:
why is this annotation required here?

Reply:
Actually @Kuldeepkumawat144 made the function less redundant by having the definition in a single place and using it everywhere. Previously also this annotation was [there](https://github.com/nammayatri/nammayatri/pull/8147/files/103d4052517e9e6f419d8bd512861a64dff83143#diff-adfb24e0218c97d2ec5f921887089cb7153ab87d3f6c8dd319042f40bb73d707L238) before this check, can we keep it that way for now, in later PR's we can remove this if tested properly.